pywren/wrenhandler.py

```diff
@@ -87,8 +96,12 @@ def download_runtime_if_necessary(s3_client, runtime_s3_bucket, runtime_s3_key):
     condatar = tarfile.open(
         mode="r:gz",
         fileobj=wrenutil.WrappedStreamingBody(res['Body'], res['ContentLength']))
-
-    condatar.extractall(runtime_etag_dir)
+    try:
+        condatar.extractall(runtime_etag_dir)
+    except:
```

Comment:
Is there a particular exception that indicates the tarball was too big ? I guess there could be other errors like corrupt runtimes etc.

Reply:
Good question, I will dig further!